### PR TITLE
Update application.html

### PR DESCRIPTION
**Pull request by Ruchi-here**
Update application.html based on MLH guidelines

### CODE REVIEW

**Comment by gregv** [comment]
@AndrewHUNGNguyen  - heads up on this one - it adds a few more fields to the database that we need for our MLH sponsorship

**Comment by Ruchi-here** [comment]
image
image
image
image
image

**Comment by Ruchi-here** [comment]
@gregv @AndrewHUNGNguyen if we are changing the main database for storing the first and last name then we have to ensure on the confirmation page that only the first and last name from the main database is being used for confirming the name instead of the existing one

File: app/client/views/application/application.html [hunk 1]
```diff
@@ -18,48 +18,2533 @@
...
             <div id="school" class="ui search">
               <div class="ui input">
-                <input
-                  class="prompt"
-                  type="text"
-                  placeholder="School"
-                  name="school"
-                  ng-model="user.profile.school"
-                />
+                <select name="school" ng-model="user.profile.school">
```
Comment (AndrewHUNGNguyen) [inline app/client/views/application/application.html:46]: The original application file gets all of its dropdowns from a csv file located in [app/client/assets](https://github.com/opportunity-hack/quill/blob/master/app/client/assets/schools.csv) folder that applicationCtrl.js uses and parses School names from. Therefore you can update this csv file to contain all MLH-recognized schools.

File: app/client/views/application/application.html [hunk 2]
```diff
@@ -18,48 +18,2533 @@
...
+              <option value="code_school">Code School / Bootcamp</option>
+              <option value="vocational_program">Other Vocational / Trade Program or Apprenticeship</option>
+              <option value="post_doctorate">Post Doctorate</option>
+              <option value="not_a_student">I’m not currently a student</option>
+              <option value="prefer_not_to_answer">Prefer not to answer</option>
+            </select>
+          </div>
+
+          <div class="field">
+            <label>Country of Residence</label>
```
Comment (AndrewHUNGNguyen) [inline app/client/views/application/application.html:2296]: Apply the same logic that is being done for rendering the schools by putting all countries into a csv file and having a function that will parse all these countries as drop-down options.

**Comment by AndrewHUNGNguyen** [comment]
@gregv and @Ruchi-here I am so sorry that I was negligent of the fact that I was requested to review this PR. 

Here are two issues that I see:
- The many option values that are being added to this html file makes this file hard to read and maintain.
- Since the original way of adding schools as a dropdown in an application parses valid schools as a .csv file, the MLH recognized schools WILL NOT be included in the schools dropdown.

**Review by AndrewHUNGNguyen — CHANGES_REQUESTED** [review]

**Review by gregv — COMMENTED** [review]
We want to keep Name, Age, Level of Study (and not School and not Country of Residence)

**Pull request closed without merging**